eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -36,7 +41,7 @@ function Submit-Request($filePath, $packageName, $packageType)
     $query.Add('commitSha', $CommitSha)
     $query.Add('repoName', $repoName)
     $query.Add('pullRequestNumber', $PullRequestNumber)
-    $query.Add('packageName', $packageName)
+    $query.Add('packageName', $fullPackageName)
```

Comment:
@praveenkuttappan can we merge this change directly or any change needs to make in API view server side?

CC: @helen229 @AlitzelMendez

Reply:
This looks fine to me.